toolkit/tools/pkg/imagecustomizerlib/extractpartitions.go

```diff
@@ -20,29 +26,62 @@ func extractPartitions(imageLoopDevice string, outDir string, basename string, p
 		return err
 	}
 
+	// Define magicNumber and frameSize for skippable frames
+	magicNumber := uint32(0x184D2A50)
+	frameSize := uint32(16)
```

Comment:
16 seems to be hardcoded throughout the code added. Is there a way to eliminate this redundancy and define and reuse a single global constant?

Reply:
I added it as a global constant and will replace all locations where it is hardcoded, but should I replace locations where it is a part of a function definition parameter (i.e. the byte array parameters of functions like `createSkippableFrameMetadata` and `createSkippableFrameMetadata`?

Reply:
Can you make them variable length?

Reply:
Updated the code to use the global constants and not hardcoded values